docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,80 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM fileset
```

Comment:
This is part of 5.2
`GET /candidates/$CandID/$VisitLabel/dicoms/$Tarname` and `PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename` are the same endpoint using different methods.

Please remove the `Only 'GET' is currently supported.` and the 5.3 header

Reply:
Great, I will make the change.